modules/qna/src/views/full/Components/TextareaList.tsx

```diff
@@ -0,0 +1,109 @@
+import { Button } from '@blueprintjs/core'
+// @ts-ignore
```

Comment:
I'm sure those will go away but if you forgot, then you got this comment

Reply:
Yep, will be removed in a future PR when I implement the new content form/display